<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1428</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArduinoDue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddComponent">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Execute">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="ArduinoPlane">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>731</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="UserCodeSection">
    <property name="geometry">
     <rect>
      <x>803</x>
      <y>80</y>
      <width>571</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>30</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1428</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
